ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/daemon-terminattr.j2

```diff
@@ -1,22 +1,23 @@
-{% if daemon_terminattr is defined and daemon_terminattr is not none %}
+{# eos - TerminAttr #}
+{% if daemon_terminattr is arista.avd.defined %}
 !
 {# Create list of non CVaaS server #}
-{% set cvp_on_prem = namespace() %}
-{% set cvp_on_prem.ips = [] %}
-{% set cvaas = namespace() %}
-{% set cvaas.ips = [] %}
-{% for cvp_ip in daemon_terminattr.ingestgrpcurl.ips %}
+{%    set cvp_on_prem = namespace() %}
+{%    set cvp_on_prem.ips = [] %}
+{%    set cvaas = namespace() %}
+{%    set cvaas.ips = [] %}
+{%    for cvp_ip in daemon_terminattr.ingestgrpcurl.ips %}
 {%       if "arista.io" not in cvp_ip %}
 {%             do cvp_on_prem.ips.append( cvp_ip ) %}
 {%       elif "arista.io" in cvp_ip %}
 {%             do cvaas.ips.append( cvp_ip ) %}
-{%    endif %}
-{% endfor %}
+{%       endif %}
+{%    endfor %}
 daemon TerminAttr
-{% if cvp_on_prem.ips|length > 0 %}
+{%    if cvp_on_prem.ips|length > 0 %}
    exec /usr/bin/TerminAttr -ingestgrpcurl={% for cvp_ip in cvp_on_prem.ips %}{{ cvp_ip }}:{{ daemon_terminattr.ingestgrpcurl.port }}{% if not loop.last %},{% endif %}{% endfor %} -cvcompression=gzip -ingestauth=key,{{ daemon_terminattr.ingestauth_key }} -smashexcludes={{ daemon_terminattr.smashexcludes }} -ingestexclude={{ daemon_terminattr.ingestexclude }}{% if daemon_terminattr.ingestvrf is ne 'default'%} -ingestvrf={{ daemon_terminattr.ingestvrf }}{% endif %}{{ " -disableaaa" if daemon_terminattr.disable_aaa | arista.avd.default(false) == true else "" }} -taillogs
```

Comment:
With the size of this line, I would go with cli approach:

Pseudo code example:

```
{% set terminattr_cli = "exec /usr/bin/TerminAttr -ingestgrpcurl=" %}
{% for cvp_ip in cvp_on_prem.ips %}
{% set terminattr_cli = terminattr_cli ~ cvp_ip %}
...
{% endfor %}
{{ terminattr_cli }}
```

Reply:
Fixed in 2ec9dff9c3ea3d7a1fb8fa708c868ad3ed84f16d